specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/examples/CosmosDBSqlAcceleratorDelete.json

```diff
@@ -0,0 +1,12 @@
+{
+  "parameters": {
+    "accountName": "ddb1",
+    "resourceGroupName": "rg1",
+    "api-version": "2020-06-01-preview",
+    "subscriptionId": "subid"
+  },
+  "responses": {
+    "202": {},
```

Comment:
For create we are saying 200 even though it's asynchronous, should we do the same over here? How did we normally express asynchronous operations?

Reply:
Any one of the valid responses is fine in the examples. 

The fact that 202 is defined as a valid response, is the indication that it is an async operation.